zha/zigbee/cluster_handlers/__init__.py

```diff
@@ -624,6 +631,46 @@ async def write_attributes_safe(
                     f"Failed to write attribute {name}={value}: {record.status}",
                 )
 
+    async def _discover_attributes_all(
```

Comment:
Wan't sure if this should go here in ZHA or in ZIGPY. The cache of unsupported attributes does exist in ZIGPY.